tap_pendo/streams.py

```diff
@@ -815,7 +825,7 @@ class TrackTypes(Stream):
     replication_method = "INCREMENTAL"
     replication_key = "lastUpdatedAt"
 
-    def get_body(self):
+    def get_body(self, bookmark):
```

Comment:
Why do we update the get_body function of many streams?

Reply:
We observed that Accounts, Features, TrackTypes, Guides, and Pages are using the same sync function and also requesting all data so added a filter to ask for data greater than the respective bookmark.